src/PolyhedralGeometry/Cone/properties.jl

```diff
@@ -55,6 +57,68 @@ julia> matrix(ZZ, rays(P))
 ```
 """
 rays(C::Cone{T}) where T<:scalar_types = rays(RayVector{T}, C)
+_rays(C::Cone{T}) where T<:scalar_types = _rays(RayVector{T}, C)
+
+
+@doc Markdown.doc"""                                                 
+    rays_modulo_lineality(as, C::Cone)
+
+Return the rays of the cone of `C` up to lineality as a `NamedTuple` with two
+iterators. If `C` has lineality `L`, then the iterator `rays_modulo_lineality`
+iterates over representatives of the rays of `C/L`. The iterator
+`lineality_basis` gives a basis of the lineality space `L`.
+
+# Examples
+For a pointed cone we get the usual rays:
+```jldoctest
+julia> C = positive_hull([1 0; 0 1])
+A polyhedral cone in ambient dimension 2
+
+julia> rays(C)
+2-element SubObjectIterator{RayVector{fmpq}}:
+ [1, 0]
+ [0, 1]
+
+julia> RML = rays_modulo_lineality(C)
+(rays_modulo_lineality = RayVector{fmpq}[[1, 0], [0, 1]], lineality_basis = RayVector{fmpq}[])
+
+julia> RML.rays_modulo_lineality
+2-element SubObjectIterator{RayVector{fmpq}}:
+ [1, 0]
+ [0, 1]
+
+julia> RML.lineality_basis
+0-element SubObjectIterator{RayVector{fmpq}}
+```
+If the cone has lineality, the second iterator iterates over a basis for the
+space of lineality.
```

Comment:
It would be useful to articulate the difference of the below C to the one above

Reply:
Done: https://github.com/oscar-system/Oscar.jl/pull/1911/commits/8b3861f8608b33f26e35bd91090d69cb8ac0c0db